ideascube/configuration/tests/test_main_api.py

```diff
@@ -77,6 +77,59 @@ def test_get_default_configuration(monkeypatch):
     assert get_config('tests', 'setting1') == 42
 
 
+def test_reset_configuration(monkeypatch, user):
+    monkeypatch.setattr(
+        'ideascube.configuration.registry.REGISTRY', {
+            'tests': {
+                'setting1': {'type': str},
+                'setting2': {'type': int},
+            },
+            'tests2': {
+                'setting3': {'type': list},
+            },
+        })
+
+    set_config('tests', 'setting1', 'A string', user)
+    set_config('tests', 'setting2', 42, user)
+    set_config('tests2', 'setting3', ['A', 'list'], user)
```

Comment:
That's already verified in the tests for `set_config`, so it's not necessary IMHO.

In any case, it doesn't hurt, so I'll add it.